# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xaw3d/redhat.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xc/lib/Xaw3d/XawI18n.h.orig	Wed Jun 20 14:18:34 2001
+++ ./xc/lib/Xaw3d/XawI18n.h	Wed Jun 20 14:18:50 2001
@@ -27,9 +27,13 @@
 
 ********************************************************/
 
+#if 0
 #ifdef HAS_WCTYPE_H
 #include <wctype.h>
+#if !defined(__linux__)
 #include <widec.h>
+#endif
+#endif
 #define wcslen(c) wslen(c)
 #define wcscpy(d,s) wscpy(d,s)
 #define wcsncpy(d,s,l) wsncpy(d,s,l)
--- ./xc/lib/Xaw3d/Imakefile.orig	Wed Jun 20 14:14:35 2001
+++ ./xc/lib/Xaw3d/Imakefile	Wed Jun 20 14:18:50 2001
@@ -10,27 +10,6 @@
 #define IncSubdir X11
 #define IncSubSubdir Xaw3d
 
-#ifdef LinuxArchitecture
-JUMP_ROOT_Xaw3d = $(AWIDGETSRC)3d
-JUMP_IS_HOST_Xaw3d = YES
-JUMP_STUBLIBS_Xaw3d = libXaw3d.sa
-JUMP_SUBDIRS_Xaw3d = $(JUMP_ROOT_Xt)
-JUMP_DIR_Xaw3d = $(JUMP_ROOT_Xaw3d)/shared
-JUMP_VARS_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_vars
-JUMP_FUNCS_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_funcs
-JUMP_IGNORE_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_ignore
-JUMP_EXPORT_Xaw3d = $(JUMP_DIR_Xaw3d)/jump.vars
-JUMP_IMPORT_Xaw3d = $(JUMP_EXPORT_Xt) $(JUMP_EXPORT_X11)
-JUMP_ADDRESS_Xaw3d = 0x60300000
-JUMP_JUMPTABLESIZE_Xaw3d = 0x4000
-JUMP_GOTSIZE_Xaw3d = 4096
-JUMP_STUBNAMES_Xaw3d = libXaw3d
-JUMP_STUBS_IMPORT_Xaw3d = $(JUMP_STUBS_IMPORT_X11)
-JUMP_LDLIBS_Xaw3d = $(JUMP_ROOT_Xt)/libXt.sa $(JUMP_ROOT_Xt)/libXmu.sa \
-        $(JUMP_ROOT_Xt)/libXext.sa $(JUMP_ROOT_X11)/libX11.sa \
-        $(JUMP_LDLIBS_libc)
-#endif
-
 #ifdef SharedXawReqs
 REQUIREDLIBS = SharedXawReqs
 #endif
@@ -41,7 +20,7 @@
 
 #  EXTRA_DEFINES = -DARROW_SCROLLBAR -DUSEGRAY
 
-#  EXTRA_INCLUDES = -I.
+EXTRA_INCLUDES = -I.
 
 HEADERS = \
 	AllWidgets.h \
